ros/processors/archive_processor.py

```diff
@@ -0,0 +1,78 @@
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+import requests
+
+from insights import extract, run, rule, make_metadata
```

Comment:
Can we directly import these modules from insights? I was looking `rule` and `make_metadata` under `insights.core.plugins` and `extract` under `insights.core.archives`